Prace_domowe/Praca_domowa4/Pawel_Wojciechowski/pd4_Pawel_Wojciechowski.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>To nie jest najlepsze przedstawienie. Przydałyby się jakieś linie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>